### PR TITLE
Upgrade grpc

### PR DESCRIPTION
**Pull request by woksin**
## Summary

Updates grpc packages and uses a single grpc channel for all connections

### Changed

- Updated grpc version to 2.43.0
- Updated contracts version
- Uses one GrpcChannel for all gRPC connections. This is a potential breaking change for netcoreapp 3.1 since it does not allow multiple concurrent http2 connections for one GrpcChannel meaning that if there are 100 or more event processors in a netcoreapp 3.1 application it will start hanging. 

### Removed

- Removed the Contrib.M1 Grpc native executable for arm64. This is no longer needed with the new .NET Grpc packages

### CODE REVIEW

**Comment by woksin** [comment]
Retrigger build once Contracts has been released

File: Source/Events.Processing/EventProcessors.cs [hunk 1]
```diff
@@ -50,6 +52,12 @@ public class EventProcessors : IEventProcessors
         where TRequest : class
         where TResponse : class
     {
+#if NETCOREAPP3_1
+        if (++_eventProcessoCounter == 100)
```
Comment (jakhog) [inline Source/Events.Processing/EventProcessors.cs]: Lets make this warning appear more than once:
```suggestion
        if (++_eventProcessoCounter >= 100)
```

File: Source/Services/ICanCallADuplexStreamingMethod.cs [hunk 2]
```diff
@@ -21,5 +21,5 @@ public interface ICanCallADuplexStreamingMethod<TClientMessage, TServerMessage>
     /// <param name="channel">The <see cref="Channel"/> to use to connect to the server.</param>
```
Comment (jakhog) [inline Source/Services/ICanCallADuplexStreamingMethod.cs]:
```suggestion
    /// <param name="channel">The <see cref="ChannelBase"/> to use to connect to the server.</param>
```

File: Source/Services/ICanCallAServerStreamingMethod.cs [hunk 3]
```diff
@@ -22,5 +22,5 @@ public interface ICanCallAServerStreamingMethod<in TClientMessage, TServerMessag
     /// <param name="channel">The <see cref="Channel"/> to use to connect to the server.</param>
```
Comment (jakhog) [inline Source/Services/ICanCallAServerStreamingMethod.cs]:
```suggestion
    /// <param name="channel">The <see cref="ChannelBase"/> to use to connect to the server.</param>
```

File: Source/Services/ICanCallAUnaryMethod.cs [hunk 4]
```diff
@@ -22,5 +23,5 @@ public interface ICanCallAUnaryMethod<TClientMessage, TServerMessage>
     /// <param name="channel">The <see cref="Channel"/> to use to connect to the server.</param>
```
Comment (jakhog) [inline Source/Services/ICanCallAUnaryMethod.cs]:
```suggestion
    /// <param name="channel">The <see cref="ChannelBase"/> to use to connect to the server.</param>
```

File: Source/Aggregates/Internal/AggregateRootsRegisterAliasMethod.cs [hunk 5]
```diff
@@ -4,6 +4,7 @@
 using Dolittle.Runtime.Aggregates.Contracts;
 using Dolittle.SDK.Services;
 using Grpc.Core;
+using Grpc.Net.Client;
```
Comment (jakhog) [inline Source/Aggregates/Internal/AggregateRootsRegisterAliasMethod.cs]:
```suggestion
```

File: Samples/Tutorials/EventHorizon/Environment/docker-compose.yml [hunk 6]
```diff
@@ -24,6 +26,8 @@ services:
       - ./producer-platform.json:/app/.dolittle/platform.json
       - ./producer-resources.json:/app/.dolittle/resources.json
       - ./producer-event-horizon-consents.json:/app/.dolittle/event-horizon-consents.json
+    environment:
+      - Logging__Console__FormatterName=""
```
Comment (jakhog) [inline Samples/Tutorials/EventHorizon/Environment/docker-compose.yml]: This shouldn't be necessary when we set it in the docker image?

**Review by jakhog — CHANGES_REQUESTED** [review]
Let's get rid of the build warnings :)

File: Source/Events.Processing/EventProcessors.cs (hunk 1 above)
Comment (woksin) [reply Source/Events.Processing/EventProcessors.cs]: 😄

File: Samples/Tutorials/EventHorizon/Environment/docker-compose.yml (hunk 6 above)
Comment (woksin) [reply Samples/Tutorials/EventHorizon/Environment/docker-compose.yml]: Yes, shouldn't be necessary

**Comment by woksin** [comment]
All build warnings should be gone now :)

**Review by jakhog — APPROVED** [review]

**Pull request merged**